src/Facades/Excel.php

```diff
@@ -12,7 +12,7 @@
 
 /**
  * @method static BinaryFileResponse download(object $export, string $fileName, string $writerType = null, array $headers = [])
- * @method static bool store(object $export, string $filePath, string $disk = null, string $writerType = null, $diskOptions = [])
+ * @method static bool store(object $export, string $filePath, string $diskName = null, string $writerType = null, $diskOptions = [])
```

Comment:
ther are some more places to fix this: 
<img width="745" alt="image" src="https://github.com/SpartnerNL/Laravel-Excel/assets/2960938/ea8a556e-11e7-4fd5-b294-28606f9cb89a">

Reply:
However changing it in other places like Exportable would make it a breaking change, as someone doing the following

`(new MyExport)->store('path', disk: 'public')` 

would get an error if we can it to diskname.

I think for now it would probably be best to just update the facade docblock and leave the rest until a major release with upgrade notes.

Reply:
> ther are some more places to fix this: <img alt="image" width="745" src="https://user-images.githubusercontent.com/2960938/279173337-ea8a556e-11e7-4fd5-b294-28606f9cb89a.png">

Except that in other places in the source code (non Facade) `$disk` is used, `store` is the exception

Reply:
What if we leave everything as is, and just add a fallback option to Excel@store, so `disk: 'name'` would work as well via the facade.

```
public function store($export, string $filePath, string $diskName = null, string $writerType = null, $diskOptions = [], string $disk =null)
```

That way no breaking change, but ensuring consistency with the facade and the other locations. (Honestly absolutely no idea why it isn't consistent, it of course didn't matter before named params)

Reply:
Sounds fine to me, then we can drop one of the two in a major version

Reply:
Yes exactly, can you update your PR?